[2024-11-18T19:03:29.045+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-18T19:03:29.643+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_world_dag.accurate scheduled__2024-11-17T00:00:00+00:00 [queued]>
[2024-11-18T19:03:30.040+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_world_dag.accurate scheduled__2024-11-17T00:00:00+00:00 [queued]>
[2024-11-18T19:03:30.041+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-18T19:03:30.908+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): accurate> on 2024-11-17 00:00:00+00:00
[2024-11-18T19:03:31.326+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=84) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-18T19:03:31.507+0000] {standard_task_runner.py:72} INFO - Started process 117 to run task
[2024-11-18T19:03:31.406+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'hello_world_dag', 'accurate', 'scheduled__2024-11-17T00:00:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/hello_world_dag.py', '--cfg-path', '/tmp/tmpry8nycgv']
[2024-11-18T19:03:31.535+0000] {standard_task_runner.py:105} INFO - Job 36: Subtask accurate
[2024-11-18T19:04:21.145+0000] {job.py:229} INFO - Heartbeat recovered after 54.59 seconds
[2024-11-18T19:04:21.369+0000] {local_task_job_runner.py:266} INFO - Task exited with return code -15
[2024-11-18T19:04:21.552+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
